distribution/changelog.txt

```diff
@@ -44,6 +44,7 @@
 - Fix: [#6423] Polish characters now correctly drawn when using the sprite font.
 - Fix: [#6445] Guests' favourite ride improperly set when importing from RCT1 or AA.
 - Fix: [#6452] Scenario text cut off when switching between 32 and 64-bit builds.
+- Fix: [#6470] Predefined title sequence naming issues
```

Comment:
Since this isn't a user-facing bug, I think it doesn't need to be listed in the changelog.

Reply:
Changelog entry removed